docs/SRS/SRS.tex

```diff
@@ -142,31 +189,129 @@ \subsection{Style Requirements}
 
 \section{Usability and Humanity Requirements}
 \subsection{Ease of Use Requirements}
-\lips
+\begin{enumerate}
+  \item \textbf{Requirement:} The tool must have an intuitive user interface that simplifies navigation and functionality.
+  \begin{itemize}[label={}]
+      \item \textbf{Fit Criteria:}  Users should be able to complete key tasks (e.g., parsing code, configuring settings) within three clicks or less.
+  \end{itemize}
+  \item \textbf{Requirement:} The tool should provide clear and concise prompts for user input.
+  \begin{itemize}[label={}]
+      \item \textbf{Fit Criteria:} At least 90\% of test users should report that prompts are straightforward and guide them effectively through the process.
+  \end{itemize}
+\end{enumerate}
 \subsection{Personalization and Internationalization Requirements}
-\lips
+\begin{enumerate}
+  \item \textbf{Requirement:}The tool should allow users to customize settings to match their preferences (e.g., refactoring styles, detection sensitivity).
+  \begin{itemize}[label={}]
+      \item \textbf{Fit Criteria:}  Users should be able to save and load custom configurations easily.
+  \end{itemize}
+  \item \textbf{Requirement:} User guide must be available in French and English.
+  \begin{itemize}[label={}]
+      \item \textbf{Fit Criteria:} French and english installation and use instructions available.
+  \end{itemize} 
+\end{enumerate}
 \subsection{Learning Requirements}
-\lips
+\begin{enumerate}
+  \item \textbf{Requirement:} The tool must have an availible youtube video demonstrating installation.
```

Comment:
We actually gonna do this?

Reply:
we can talk about it in class. Someone has to install it at some point this is an easy screen record.